firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java

```diff
@@ -54,6 +55,13 @@ public List<Component<?>> getComponents() {
             .add(Dependency.requiredProvider(TransportFactory.class))
             .factory(FirebasePerfRegistrar::providesFirebasePerformance)
             .build(),
+        Component.builder(FirebasePerfEarly.class)
+            .name(LIBRARY_NAME)
```

Comment:
are we ok to have the same name for this component? i.e. it could be confusing when looking at startup traces. Given that FirebasePerformance itself is not eager, might make sense to make it unnamed, wdyt?

Reply:
As a shim only giving the name to the early component makes sense, but it was easy enough to just give them unique names.